modules/liberty/src/main/java/org/testcontainers/containers/liberty/LibertyAdapter.java

```diff
@@ -112,4 +112,8 @@ public ImageFromDockerfile getDefaultImage(File appFile) {
         return image;
     }
 
+    @Override
+    public String getReadinessPath() {
+        return "health/ready";
```

Comment:
lets have this return an empty Optional for now, because /health/ready will only be available if the user has enabled the mpHealth-2.0 feature in their server.xml, which isn't enabled in a lot of cases